Scheduler add getting started (#1484)
DevExpress/devextreme-documentation: concepts/05 UI Components/Scheduler/00 Getting Started with Scheduler/30 Enable Adaptive Mode.md

> The **Scheduler** can adapt its interface to small screens. To enable this behavior, set the [adaptivityEnabled](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#adaptivityEnabled) property to **true**. A full list of adaptations is given in the [Adaptability](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Scheduler/Adaptability/) demo description.

Reviewer: `A full list of adaptations is given in the [Adaptability](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Scheduler/Adaptability/) demo description.`

Are you sure "adaptations" is the correct technical term in this context? (Just checking.) Or, based on my understanding of the demo, would it be better to say "adaptability features"? For example, could we say:

``

```suggestion
The **Scheduler** can adapt its interface to small screens. To enable this behavior, set the [adaptivityEnabled](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#adaptivityEnabled) property to **true**. See the following demo description for a full list of adaptability features: [Adaptability](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Scheduler/Adaptability/).
```
